src/containers/RegisterRouting/components/NameForm.jsx

```diff
@@ -1,39 +1,36 @@
 import {Link} from "react-router-dom";
 import {useRef} from "react";
+import UserService from "../../../services/UserService";
+
+//todo: add validation for input fields, disable button
 
 function NameForm(props) {
+    const inputNameRef = useRef();
+
     const onClickHandler = (event) => {
         props.setName(saveName);
     };
 
-    const inputNameRef = useRef();
-
-    const saveName = () => {
-        const nameFromInput = inputNameRef.current.value;
+   const saveName = () => {
+       const nameFromInput = inputNameRef.current.value;
+       console.log(typeof nameFromInput, nameFromInput);
```

Comment:
You can leave console logs, when you logging some kind of errors of useful info for validation. If you need this for debugging only, don't forget to remove it before merge